src/cookbook/design/themes.md

```diff
@@ -53,34 +88,45 @@ MaterialApp(
 );
 ```
 
-See the [`ThemeData`][] documentation to see all of
-the colors and fonts you can define.
+Most `ThemeData` objects include two properties.
+
+1. [`colorScheme`][] defines colors.
+1. [`textTheme`][] defines text styling.
+
+[`colorScheme`]: {{site.api}}/flutter/material/ThemeData/colorScheme.html
+[`textTheme`]: {{site.api}}/flutter/material/ThemeData/textTheme.html
+
+To learn what colors and fonts you can define,
+check out the [`ThemeData`][] documentation.
 
-## Themes for part of an application
+## Override a theme
 
-To override the app-wide theme in part of an application,
-wrap a section of the app in a `Theme` widget.
+To override the overall theme in part of an app,
+wrap that section of the app in a `Theme` widget.
 
-There are two ways to approach this: creating a unique `ThemeData`,
-or extending the parent theme.
+You can override a theme in two ways:
+
+1. Create a unique `ThemeData` instance.
+2. Extend the parent theme.
 
 {{site.alert.note}}
   To learn more, watch this short Widget of the Week video on the Theme widget:
 
   <iframe class="full-width" src="{{site.youtube-site}}/embed/oTvQDJOBXmM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 {{site.alert.end}}
 
-### Creating unique `ThemeData`
+### Set a unique `ThemeData` instance
 
-If you don't want to inherit any application colors or font styles,
-create a `ThemeData()` instance and pass that to the `Theme` widget.
+If you want a component of your app to ignore the overall theme,
+create a `ThemeData()` instance.
```

Comment:
FWIW, this is perfect. ThemeData instance and Theme widget is exactly correct terminology.